### PR TITLE
Allow storages.backends.azure_storage.AzureStorage as value DEFAULT_F…

### PR DESCRIPTION
**Pull request by makesse**
Allow Azure blob storage as back-end storage for Pulp and provide the documentation on how to configure it.

### CODE REVIEW

File: docs/installation/storage.rst [hunk 1]
```diff
@@ -67,3 +67,38 @@ Configuring Pulp
...
+
+Configuring Pulp
+----------------
+
+  To have Pulp use Azure Blob storage, you'll need to install the optional django-storages[azure]
+  package in the pulp virtual environment::
+
+      pip install django-storages[azure]
+
+  Nest you'll need to set the ``DEFAULT_FILE_STORAGE`` to
```
Comment (daviddavis) [inline docs/installation/storage.rst]: s/Nest/Next/

File: docs/installation/storage.rst [hunk 2]
```diff
@@ -67,3 +67,38 @@ Configuring Pulp
...
+  ``storages.backends.azure_storage.AzureStorage`` in your pulp settings, and configure following
+  parameters n your pulp settings file (for a comprehensive overview of all possible options for
```
Comment (daviddavis) [inline docs/installation/storage.rst]: parameters in

File: docs/installation/storage.rst [hunk 3]
```diff
@@ -67,3 +67,38 @@ Configuring Pulp
...
+  the Azure Blob storage backend see the `django-storages[azure] documents
+  <https://django-storages.readthedocs.io/en/latest/backends/azure.html>`::
```
Comment (daviddavis) [inline docs/installation/storage.rst]: Missing a close `)`

**Comment by daviddavis** [comment]
This is excellent :+1: 

We want to advertise this in our CHANGELOG. Mind [filing an issue in redmine](https://pulp.plan.io/issues/new) and [creating an entry in CHANGES](https://docs.pulpproject.org/en/master/nightly/contributing/git.html#changelog-update)?

**Comment by makesse** [comment]
@daviddavis : I have created issue https://pulp.plan.io/issues/5954 for this, and added the entry to CHANGES. Do you want to change the commit message to include the issue?

File: docs/installation/storage.rst [hunk 4]
```diff
@@ -67,3 +67,38 @@ Configuring Pulp
...
+  To have Pulp use Azure Blob storage, you'll need to install the optional django-storages[azure]
+  package in the pulp virtual environment::
+
+      pip install django-storages[azure]
+
+  Next you'll need to set the ``DEFAULT_FILE_STORAGE`` to
+  ``storages.backends.azure_storage.AzureStorage`` in your pulp settings, and configure following
+  parameters in your pulp settings file (for a comprehensive overview of all possible options for
+  the Azure Blob storage backend see the `django-storages[azure] documents
+  <https://django-storages.readthedocs.io/en/latest/backends/azure.html>`)::
```
Comment (daviddavis) [inline docs/installation/storage.rst]:
```suggestion
  <https://django-storages.readthedocs.io/en/latest/backends/azure.html>_`)::
```

**Comment by daviddavis** [comment]
Everything looks great. One link needs to be fixed. If you accept my suggestion, I can merge this. 

Thanks @makesse!

**Comment by daviddavis** [comment]
To answer your question: the commit message should preferably include `fixes #5954` instead of `[noissue]`.

**Pull request merged**